WINDOWS.md

```diff
@@ -0,0 +1,144 @@
+# Windows Installaion
+
+There are two main methods you can use to setup Fabrik on a Windows system with a virtual environment.
+Firstly, you can use WSL, the Windows Subsystem for Linux, which is by far the easier option.
+To install Fabrik with WSL, all you need to do is simply install a distro of your choice through the Windows Store, and then follow the Linux installation instructions in the README.
+The alternative is to install natively through the Win32 subsystem, which is significantly more complicated, but an option if the WSL is not available.
+
+### WSL Notes
+
+The setup is exactly the same as other linux systems. The Ubtuntu disto has been tested and confirmed to work.
+It is iportant Fabrik is setup in `/mnt/c/...`, rather than the Linux user profile directory, because editing of the WSL subsystem from the Win32 subsystem can leading to file corruption. 

```

Comment:
```suggestion
It is important that Fabrik is setup in `/mnt/c/...`, rather than the Linux user profile directory, because editing of the WSL subsystem from the Win32 subsystem can leading to file corruption. 
```
Minor typo